# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: target/mnemosyne/pkg_patch/nginx.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./conf/nginx.conf.orig	2010-09-09 19:05:20.000000000 +0000
+++ ./conf/nginx.conf	2010-09-09 19:06:00.000000000 +0000
@@ -1,7 +1,9 @@
 
-#user  nobody;
+user http http;
 worker_processes  1;
+daemon off;
 
+error_log	/dev/stderr info;
 #error_log  logs/error.log;
 #error_log  logs/error.log  notice;
 #error_log  logs/error.log  info;
